helpers.go

```diff
@@ -300,7 +300,6 @@ func updatePatchCmd(zypperCmd string, ctx *cli.Context) {
 		log.Println("Cannot add image details to zypper-docker cache")
 		log.Println("This will break the \"zypper-docker ps\" feature")
 		log.Println(err)
-		exitWithCode(1)
```

Comment:
Initially I wasn't 100% sure about dropping this exit code. However, thinking about zypper-docker automated usage, I accept the change. A user will get the warning message